### PR TITLE
Migrate to bitvec 0.22

### PR DESCRIPTION
**Pull request by str4d**
MSRV is now 1.51.0.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -45,5 +45,15 @@ required-features = ["groth16"]
...
 
+[[bench]]
+name = "slow"
+harness = false
+
 [badges]
 maintenance = { status = "actively-developed" }
+
+[patch.crates-io]
+bls12_381 = { git = "https://github.com/nuttycom/bls12_381.git", branch = "bitvec-0.22" }
```
Comment (str4d) [inline Cargo.toml:56]: Once https://github.com/zkcrypto/bls12_381/pull/62 is merged:
```suggestion
bls12_381 = { git = "https://github.com/zkcrypto/bls12_381.git", branch = "main" }
```

**Comment by str4d** [comment]
Before the upgrade (the second commit in this PR):
```
Gnuplot not found, using plotters backend
Benchmarking multiexp/65536: Warming up for 3.0000 s
Warning: Unable to complete 100 samples in 5.0s. You may wish to increase target time to 28.5s, or reduce sample count to 10.
multiexp/65536          time:   [288.62 ms 291.94 ms 295.83 ms]                           
Found 12 outliers among 100 measurements (12.00%)
  6 (6.00%) high mild
  6 (6.00%) high severe
```
After the upgrade:
```
Benchmarking multiexp/65536: Warming up for 3.0000 s
Warning: Unable to complete 100 samples in 5.0s. You may wish to increase target time to 20.1s, or reduce sample count to 20.
multiexp/65536          time:   [199.05 ms 199.63 ms 200.24 ms]                           
                        change: [-32.545% -31.619% -30.805%] (p = 0.00 < 0.05)
                        Performance has improved.
Found 1 outliers among 100 measurements (1.00%)
  1 (1.00%) high mild
```

**Review by daira — APPROVED** [review]
utACK

**Review by ebfull — APPROVED** [review]

File: Cargo.toml [hunk 2]
```diff
@@ -24,7 +24,7 @@ byteorder = "1"
 subtle = "2.2.1"
 
 [dev-dependencies]
-bls12_381 = "0.4"
+bls12_381 = { version = "0.4", features = ["bits"] }
```
Comment (str4d) [inline Cargo.toml:27]:
```suggestion
bls12_381 = "0.4"
```

**Pull request merged**